library/src/main/kotlin/it/krzeminski/githubactions/dsl/Concurrency.kt

```diff
@@ -0,0 +1,16 @@
+package it.krzeminski.githubactions.dsl
```

Comment:
It should rather be in the domain package:
```suggestion
package it.krzeminski.githubactions.domain
```

Reply:
done